packages/wallets/src/components/Base/WalletButton/WalletButton.tsx

```diff
@@ -31,12 +36,12 @@ const WalletButton: React.FC<WalletButtonProps> = ({
     const hasText = !!text;
 
     const buttonClassNames = classNames(
-        styles['wallets-button'],
-        isContained && styles[`wallets-button-color-${color}`],
-        styles[`wallets-button-size-${size}`],
-        styles[`wallets-button-variant-${variant}`],
-        isRounded ? styles['wallets-button-rounded-full'] : styles['wallets-button-rounded-default'],
-        isFullWidth && styles['wallets-button-full-width']
+        'wallets-button',
+        isContained && `wallets-button-color-${color}`,
+        `wallets-button-size-${size}`,
+        `wallets-button-variant-${variant}`,
+        isRounded ? 'wallets-button-rounded-full' : 'wallets-button-rounded-default',
+        isFullWidth && 'wallets-button-full-width'
```

Comment:
Alternatively we can use classNames like these thou, but its really just nitpicking ignore this if we find it harder to understand:
```suggestion
classNames('wallets-button', `wallets-button-size-${size}`, `wallets-button-variant-${variant}` {
        `wallets-button-color-${color}`: isContained,
        'wallets-button-rounded-full': isRounded,
        'wallets-button-rounded-default': !isRounded,
        'wallets-button-full-width': isFullWidth
})
```

Reply:
I dont think its works given my situation because most of the classes I am still using it